### PR TITLE
Refactors flaky ratelimit tests

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: proxy/ratelimit_test.go [hunk 1]
```diff
@@ -232,74 +92,15 @@ func TestCheckLocalRateLimit(t *testing.T) {
...
-		code, _ := request(d0)
-		if code == http.StatusTooManyRequests {
-			t.Fatal("should not be ratelimitted")
-		}
-	}
-	for i := 0; i < 10; i++ {
-		d0 := doc(128)
-		code, header := request(d0)
-		if code != http.StatusTooManyRequests {
-			t.Fatal("should be ratelimitted")
```
Comment (AlexanderYastrebov) [inline proxy/ratelimit_test.go:280]: E.g. [this failure](https://github.com/zalando/skipper/pull/1670/checks?check_run_id=1622366041) is apparenly due to slow `doc(128)` which uses `rand.Read`. Note also that doc is created but is never used

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**